docs/import/add_disaster.js

```diff
@@ -194,7 +192,12 @@ function getAssetsFromEe(states) {
         for (const state of states) {
           const dir = legacyStateDir + '/' + state;
           if (!folders.has(state)) {
+            // This will print a console error for anyone other than the gd
+            // account. Ee console seems to have the power to grant write access
+            // to non-owners but it doesn't seem to work. Sent an email to
+            // gestalt.
             ee.data.createFolder(dir, false, () => {
+              // TODO: add status bar for when this is finished.
               ee.data.setAssetAcl(dir, {all_users_can_read: true});
```

Comment:
Looks like this is deprecated, and setIamPolicy is the new hotness

Reply:
So i get a pretty gross error when trying to use setIamPolicy. I actually get this also with the gd account and pretty unclear to me what it's about. So I'm going to stick with setAclAsset for now, leave a TODO and update the email to gestalt. 

![image](https://user-images.githubusercontent.com/30670028/68885655-8f50c700-06e3-11ea-8f98-6efc2b1d8fa4.png)